corehq/apps/export/models/new.py

```diff
@@ -522,12 +522,11 @@ def get_rows(self, document, row_number, split_columns=False,
                     transform_dates=transform_dates,
                 )
                 if as_json:
-                    if isinstance(val, list):
-                        headers = col.get_headers(split_column=split_columns)
-                        for subval_ind, subval in enumerate(val):
-                            row_data[headers[subval_ind]] = "{}".format(subval)
-                    else:
-                        row_data[col.label] = "{}".format(val)
+                    for ind, header in enumerate(col.get_headers(split_column=split_columns)):
+                        if isinstance(val, list):
+                            row_data[header] = "{}".format(val[ind])
+                        else:
+                            row_data[header] = "{}".format(val)
```

Comment:
i'm a little unclear why this change was necessary. this puts the same value in multiple columns if there were multiple headers but a single value? when does that happen?

Reply:
if there is a single value there is only a single header. there isn't really an instance where there are multiple headers and one value. this just simplifies the code a bit

Reply:
unfortunately, if there is a single value, it is no longer a list...so `val[0]` doesn't work. so perhaps what we should really address is that inconsistency in the exports code!

Reply:
makes sense, thanks. tiny unrelated suggestion you can take or leave is maybe rename `ind` to `index` for readability. `ind` is a little nonstandard and for some reason makes me think of "indicator"